external_parser/log_converter.cc

```diff
@@ -123,8 +132,148 @@ void build_cb_json(std::ofstream &outfile,
     outfile << out_buffer.GetString() << std::endl;
   } catch (const std::exception &e) {
     VW::io::logger::log_error(
-        "convert events: [{}] from binary to json format failed: [{}].",
+        "convert event: [{}] from binary to json format failed: [{}].",
         interaction_data.eventId, e.what());
   }
 }
+
+void build_ccb_json(std::ofstream &outfile,
+                    joined_event::joined_event &je) {
+  std::string event_id = je.interaction_metadata.event_id;
```

Comment:
does this need to be a copy or can it be a reference? i.e. `std::string& event_id` or `const std::string& event_id`

Reply:
replaced with reference